Spring-server/src/main/java/ikpi63holding/postclient/mail/MediaTypeUtils.java

```diff
@@ -0,0 +1,23 @@
+package ikpi63holding.postclient.mail;
+
+import javax.servlet.ServletContext;
+import org.springframework.http.MediaType;
+
+public class MediaTypeUtils {
+
+    // abc.zip
+    // abc.pdf,..
+    public static MediaType getMediaTypeForFileName(ServletContext servletContext,
+            String fileName) {
+        // application/pdf
+        // application/xml
+        // image/gif, ...
```

Comment:
what's this?

Reply:
I can only guess that this is some example of media types, but it is not my component